### PR TITLE
INDY-902 test_send_schema

### PR DESCRIPTION
**Pull request by cam-parra**
Signed-off-by: Cam Parra <caeparra@gmail.com>

### CODE REVIEW

File: indy_client/test/cli/test_send_schema.py [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
 from indy_client.test.cli.constants import SCHEMA_ADDED, SCHEMA_NOT_ADDED_DUPLICATE
+from indy_node.test.api.helper import sdk_write_schema
+from plenum.test.helper import sdk_get_reply, sdk_sign_and_submit_req
 
 
-def test_send_schema_multiple_attribs(be, do, poolNodesStarted, trusteeCli):
-    be(trusteeCli)
-    do('send SCHEMA name=Degree version=1.0 keys=attrib1,attrib2,attrib3',
-       expect=SCHEMA_ADDED, within=5)
+def testSendSchemaMultipleAttribs(looper, sdk_pool_handle, sdk_wallet_trust_anchor):
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_schema.py]: Please use snake case for method and test names

File: indy_client/test/cli/test_send_schema.py [hunk 2]
```diff
@@ -1,10 +1,24 @@
+import json
+
+from indy.anoncreds import issuer_create_schema
+from indy.ledger import build_schema_request
 from indy_client.test.cli.constants import SCHEMA_ADDED, SCHEMA_NOT_ADDED_DUPLICATE
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_schema.py]: Please make sure that there are no indy_client dependencies there

File: indy_client/test/cli/test_send_schema.py [hunk 3]
```diff
@@ -1,10 +1,24 @@
+import json
+
+from indy.anoncreds import issuer_create_schema
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_schema.py]: Please move the tests into indy_node/test/... folder

File: indy_client/test/cli/test_send_schema.py [hunk 4]
```diff
@@ -1,10 +1,24 @@
...
+
+    wallet_handle, identifier = sdk_wallet_trust_anchor
+
+    schema_json, _ = sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_trust_anchor)
+    _, schema_json = looper.loop.run_until_complete(
+        issuer_create_schema(identifier, "name=Degree", "1.0", json.dumps(["attrib1","attrib2","attrib3"])))
+
+    request = looper.loop.run_until_complete(build_schema_request(identifier, schema_json))
+
+    return schema_json, \
```
Comment (ashcherbakov) [inline indy_client/test/cli/test_send_schema.py]: Each test must end with an assertion, not return.
Ideally, each test should be flat, straightforward, and consist of 3 main parts:
1) Configuration 
2) Execution
3) Assertion

**Comment by ArtObr** [comment]
Please, use flake8 or code formatting. That really can help

**Comment by cam-parra** [comment]
@AlexanderShekhovcov Please review

File: indy_node/test/api/helper.py [hunk 5]
```diff
@@ -184,3 +184,28 @@ def sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_steward):
...
+                               multi_attribute=[], name="", version=""):
+    _, identifier = sdk_wallet_steward
+
+    if multi_attribute:
+        _, schema_json = looper.loop.run_until_complete(
+            issuer_create_schema(
+                identifier, name,
+                version, json.dumps(multi_attribute)
+            ))
+    else:
```
Comment (ashcherbakov) [inline indy_node/test/api/helper.py]: Why do we need this else here?

File: indy_node/test/api/helper.py [hunk 6]
```diff
@@ -184,3 +184,28 @@ def sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_steward):
     request = looper.loop.run_until_complete(build_schema_request(identifier, schema_json))
     return schema_json, \
            sdk_get_reply(looper, sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_steward, request))[1]
+
+
+def sdk_write_schema_and_check(looper, sdk_pool_handle, sdk_wallet_steward,
+                               multi_attribute=[], name="", version=""):
```
Comment (ashcherbakov) [inline indy_node/test/api/helper.py]: I would rename the parameter here to just `attributes`

File: indy_node/test/api/helper.py (hunk 5 above)
Comment (cam-parra) [reply indy_node/test/api/helper.py]: This is for the functions that relied on this helper before my modifications.

Comment (cam-parra) [reply indy_node/test/api/helper.py]: I was incorrect. Yes I believe this if can be removed

**Pull request merged**